parsigs/parse_sig_api.py

```diff
@@ -182,6 +182,10 @@ def _get_form_from_dosage_tag(text):
     if len(splitted) == 2:
         return splitted[1]
 
+def to_singular(text):
+    # turn to singular if plural else keep as is
+    singular = inflect.engine().singular_noun(text)
```

Comment:
Add `_` in the beginning of the method (indicates it should be used only inside this package)

is calling engine a performant action? if so keep a global variable
'inflect_engine = inflect.engine()`